#Request
---
#Response
geometry_msgs/Point pixel_pose